legal-api/src/legal_api/config.py

```diff
@@ -144,9 +144,16 @@ class _Config():  # pylint: disable=too-few-public-methods
 
     NAICS_API_URL = os.getenv('NAICS_API_URL', 'https://NAICS_API_URL/api/v2/naics')
 
-    ACA_PY_ADMIN_API_URL = os.getenv('ACA_PY_ADMIN_API_URL')
-    ACA_PY_ADMIN_API_KEY = os.getenv('ACA_PY_ADMIN_API_KEY')
-    ACA_PY_ENTITY_DID = os.getenv('ACA_PY_ENTITY_DID')
+    # Digital Credential configuration values
+    TRACTION_API_URL = os.getenv('TRACTION_API_URL')
+    TRACTION_API_TOKEN = os.getenv('TRACTION_API_TOKEN')
+    TRACTION_PUBLIC_SCHEMA_DID = os.getenv('TRACTION_PUBLIC_SCHEMA_DID')
+    TRACTION_PUBLIC_ISSUER_DID = os.getenv('TRACTION_PUBLIC_ISSUER_DID')
+
+    BUSINESS_SCHEMA_NAME = os.getenv('BUSINESS_SCHEMA_NAME')
+    BUSINESS_SCHEMA_VERSION = os.getenv('BUSINESS_SCHEMA_VERSION')
+    BUSINESS_SCHEMA_ID = os.getenv('BUSINESS_SCHEMA_ID')
+    BUSINESS_CRED_DEF_ID = os.getenv('BUSINESS_CRED_DEF_ID')
```

Comment:
I'm not sure where these come from (for the BE) but probably need to work with Patrick Wei to configure these (in 1Password, I assume).

Are there different values for Dev/Test/Prod environments?

Also, where are these key values for running locally?

Reply:
Other than `BUSINESS_SCHEMA_NAME` and `BUSINESS_SCHEMA_VERSION`, there will be different values to enter for Dev/Test/Prod as they reference the Schema ID and Credential Definition for issuing the VC from different ledgers.

Reply:
For those 2, I think we need to add them to 1Password. I can do it. Or maybe Argus wants to do it. We will need the values, please.

For the other new keys, where do they come from? (We could also put them in 1Password.)

Reply:
You can add all the required config to 1pass if you are up for it Sev.  I don't have access to 1pass

Reply:
I can, but I need the values :)

Also, does the BE need a "vaults" file to pull the values from 1Pass into the env?

Reply:
i'll do some work tomorrow to get the config sorted out so you'll know what we need to update when we get the values from Akiff.  But there does seem to be a vault(bbdc) for the previous digital credential config.

Reply:
OK, I don't know how the BE gets its config from 1Pass. Probably need Patrick's help for this.

Reply:
OK, I didn't see that. Yes, this would need to be expanded. Easy.

Reply:
First, that vault doesn't allow anyone from Entity Team from accessing it. Need Patrick again.

Second, this is the "old" way to access 1Password keys. (The Entity UIs use a vaults.env file that looks completely different.) Patrick.

Reply:
What would be the easiest way to securely pass the values over to you?

Reply:
Email? severin.beauvais@gov.bc.ca

Reply:
I can provide you with the items you'll need from my dev Traction for now but I'd highly recommend that your team register its own tenant (this will be required for test and prod as is). It is very little effort to get one setup and I can walk you through that.

Reply:
Argus, are you taking care of registering tenants for all 3 environments (since we'll need our own for Dev eventually)?

Reply:
As per our conversation, Severin will be taking care of working with Akiff and SRE to get tenants registered for all envs.